ClinGen/clincoded
# Connects to #201 ft family submit

Both https://github.com/ClinGen/clincoded/issues/200 and https://github.com/ClinGen/clincoded/issues/201 are implemented in this one PR, as they’re very related.

To test:

1. From Creation Central, fill in the form for a new group. Just the minimum information works fine. Note the name you gave the group.
2. Click Save. A new page appears with the title Group Information: and the name of the group you just made.
3. Click the Edit button at the end of the title. You go back to the group you just created.
4. Change the name of the group, then click Save. You go back to the same page as before. Note that the group title has changed.
5. In the blue panel below, click No from the dropdown menu. Another gray panel appears below.
6. Now click Yes from the dropdown menu. The gray panel’s contents change.
7. Click the **Add New Family for this Group** button.
8. A family curation page appears with the title **Curate Family Information for Group:** and then the name of the group you made earlier. Fill in the form; minimal information works fine.
9. Click Save. Another new page with the title **Family Information:** and the name of the family you just made appears.
10. Click the Edit button at the end of the title. You'll go back to the family curation page.
11. Change the name of the family and click Save.
12. You go back to the same page as before, but with the new family name.
13. Click Yes from the dropdown menu. A gray panel with just a single button appears.
14. Now click No from the dropdown menu. The contents of the gray panel change.
15. Click **Return to Record Curation page**.
16. You go back to Curation Central (we need to start referring to it as the “Record Curation” page) with the article you added evidence to selected.
